dask_image/ndmeasure/__init__.py

```diff
@@ -202,6 +206,37 @@ def extrema(image, label_image=None, index=None):
     return result
 
 
+def find_objects(label_image):
```

Comment:
Suggestion: indicate here that a dask dataframe is returned instead of a list as in the ndimage version

Reply:
Good suggestion, done.